public/index.html

```diff
@@ -1,16 +1,46 @@
+    <link rel="icon" type="image/png" sizes="72x72" href="/images/android-icon-72x72.png" />
+    <link rel="icon" type="image/png" sizes="96x96" href="/images/android-icon-96x96.png" />
+    <link rel="icon" type="image/png" sizes="144x144" href="/images/android-icon-144x144.png" />
+    <link rel="icon" type="image/png" sizes="192x192" href="/images/android-icon-192x192.png" />
+    <link rel="apple-touch-icon" sizes="57x57" href="/images/apple-icon-57x57.png" />
+    <link rel="apple-touch-icon" sizes="60x60" href="/images/apple-icon-60x60.png" />
+    <link rel="apple-touch-icon" sizes="72x72" href="/images/apple-icon-72x72.png" />
+    <link rel="apple-touch-icon" sizes="76x76" href="/images/apple-icon-76x76.png" />
+    <link rel="apple-touch-icon" sizes="114x114" href="/images/apple-icon-114x114.png" />
+    <link rel="apple-touch-icon" sizes="120x120" href="/images/apple-icon-120x120.png" />
+    <link rel="apple-touch-icon" sizes="144x144" href="/images/apple-icon-144x144.png" />
+    <link rel="apple-touch-icon" sizes="152x152" href="/images/apple-icon-152x152.png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-icon-180x180.png" />
+    <link rel="icon" type="image/png" sizes="192x192" href="/images/apple-icon-precomposed.png" />
+    <link rel="icon" type="image/png" sizes="192x192" href="/images/apple-icon.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="96x96" href="/images/favicon-96x96.png" />
+    <link rel="icon" type="image/png" sizes="70x70" href="/images/ms-icon-70x70.png" />
+    <link rel="icon" type="image/png" sizes="144x144" href="/images/ms-icon-144x144.png" />
+    <link rel="icon" type="image/png" sizes="150x150" href="/images/ms-icon-150x150.png" />
+    <link rel="icon" type="image/png" sizes="310x310" href="/images/ms-icon-310x310.png" />
+
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta name="msapplication-TileImage" content="/images/ms-icon-144x144.png" />
+    <meta name="theme-color" content="#ffffff" />
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <!-- Gordon blue -->
-    <meta name="theme-color" content="#014983">
+    <meta name="theme-color" content="#014983" />
     <!--
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="manifest" href="./manifest.json" />
```

Comment:
I'm not sure if you purposefully ignored the above comment or not, but it seems like either we should obey it or we should change it to reflect what we're doing.